docs/stock/expiry.md

```diff
@@ -13,7 +13,7 @@ The stock expiry feature is disabled by default, and must be enabled via the set
 {% endwith %}
 
 !!! info "Non Expiring Stock"
-    If a Stock Item is not expected to expire, 
+    If a Stock Item is not expected to expire. 
```

Comment:
Ah, this was my omission. Could you please fix this to read:

`If a Stock Item is not expected to expire, leave the expiry date field blank, or zero`